docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```

Comment:
Why do we have this bat-file here?

Reply:
auto generated using `sphinx-quickstart` I can remove it, no big deal

Reply:
Not a problem, was just thinking if its important. We can leave it